src/runtime/link-accounts-flow-test.js

```diff
@@ -322,7 +322,44 @@ describes.realWin('LinkSaveFlow', {}, env => {
     return linkSaveFlow.start();
   });
 
+  it('should have valid LinkSaveFlow constructed with authCode', () => {
+    linkSaveFlow = new LinkSaveFlow(runtime, {authCode: 'testCode'});
+    activitiesMock.expects('openIframe').withExactArgs(
+        sinon.match(arg => arg.tagName == 'IFRAME'),
+        '$frontend$/swg/_/ui/v1/linksaveiframe?_=_',
+        {
+          _client: 'SwG $internalRuntimeVersion$',
+          publicationId: 'pub1',
+          authCode: 'testCode',
+          isClosable: true,
+        })
+        .returns(Promise.resolve(port));
+    return linkSaveFlow.start();
+  });
+
+  it('should throw error both token and authCode', () => {
+    linkSaveFlow = new LinkSaveFlow(
+      runtime, {token: 'token', authCode: 'testCode'});
+    return linkSaveFlow.start().then(() => {
+      throw new Error('must have failed');
+    }, reason => {
+      expect(() => {
+        throw reason;}).to.throw(/Both authCode and token are available/);
```

Comment:
This is because the start method itself fails, not the promise produced by this method. In other words, this is a synchronous error. To test it you can do:

```
expect(() => {
  linkSaveFlow.start();
}).to.throw(/.../);
```

Reply:
Done